src/components/Tooltip/index.native.js

```diff
@@ -1,18 +1,37 @@
+import React from 'react';
+import {View} from 'react-native';
 import PropTypes from 'prop-types';
 
 // We can't use the common component for the Tooltip as Web implementation uses DOM specific method to
 // render the View which is not present on the Mobile.
 const propTypes = {
-    children: PropTypes.element,
+    /** Styles to be assigned to the Tooltip wrapper views */
+    // eslint-disable-next-line react/forbid-prop-types
+    containerStyle: PropTypes.object,
```

Comment:
I think we prefer the styles to be `PropTypes.arrayOf(PropTypes.object)`

Reply:
and we can rename this to be `containerStyles`

Reply:
Ok! I'll change it on the non-native version too and adapt our current `Tooltip`s to the change

I guess it's better to change the `Hooverable` too since it's also receiving an object as the style?

Reply:
Yup, thanks